### ISSUE TITLE
Icons missing in Testing Window

### ISSUE THREAD

**Issue by jhhcs** [microsoft/vscode#119388]
The following is observed on a Windows 10 build 19041.867 with:
```
$ code --version
1.54.3
2b9aebd5354a3629c3aba0a5f5df49f43d6689f8
x64
```
I have the exact issue outlined in #92323, but the icons are missing exclusively from the Testing window. 

The problem does not appear to be an incorrect MIME type in the Registry: I have checked that the `Content Type` value in the registry key `HKEY_CLASSES_ROOT\.svg` has its string data set to `image/svg+xml`, and looking at the dev tools in VSCode while opening the testing tab also confirms that the correct MIME type is returned:

![recording](https://user-images.githubusercontent.com/58880204/111876640-f1889e80-899f-11eb-9d93-14d90044f54f.gif)

**Comment by connor4312** [comment]
@ma

**Comment by connor4312** [comment]
@mjbvz seems like it may be resource loading related

**Comment by mjbvz** [comment]
@connor4312 Is the testing view a webview?

**Comment by connor4312** [comment]
No, I think it's a traditional tree view

**Comment by deepak1556** [comment]
Can you run `Developer: Open Logs folder` and attach the logs. Based on the devtools response, I don't think there is an issue at the network layer.

**Comment by jhhcs** [comment]
Unfortunately, I can't just share log files from this computer publically, but I have inspected the logs from that folder and I am convinced that the only relevant one is the `main.log` file:
```
[2021-03-23 14:05:42.294] [main] [info] update#setState idle
[2021-03-23 14:05:51.493] [main] [error] file: Refused to load resource REDACTED\.vscode\extensions\ms-python.python-2021.3.658691958\resources\dark\status-unknown.svg from file: protocol (original URL: file:///REDACTED/.vscode/extensions/ms-python.python-2021.3.658691958/resources/dark/status-unknown.svg)
```
Notably, the path that has been replaced by `REDACTED` is a Junction to `%USERPROFILE%`.

**Comment by deepak1556** [comment]
The python extension error is related to https://github.com/microsoft/vscode/issues/118623#issuecomment-796472912 unrelated to this issue.

* Launch `code --log-net-log=<some-absolute-path>/netlog.json`
* perform the same steps as in your recording https://github.com/microsoft/vscode/issues/119388#issue-836851732 
* Exit the app and attach the logs.

**Comment by huettenhain** [comment]
Hey! This is @jhhcs from my main account. Redacting the net log from the work machine would have been too painful and error-prone, but I was able to reproduce the exact same problem on a different machine, and have attached the [zipped net log](https://github.com/microsoft/vscode/files/6196928/netlog.json.zip) for you.

**Comment by deepak1556** [comment]
Thanks for the log but it didn't capture any image loads, can you make sure that the relevant svg loads are triggered in the devtools before quitting the app. The above net log capturing mode will default to removing any sensitive information like cookies and auth data, and the logs will be about resources loaded by vscode app, so there is no need to redact it. If you are still worried, please send the logs to Deepak.Mohan@microsoft.com

**Comment by jhhcs** [comment]
I did actually make very sure to trigger the image loads, I had the devtools open at the same time. I can try again, but is there something I can check in the log to make sure it was captured? Should it contain the string `svg`, for example?

**Comment by deepak1556** [comment]
You can check for the svg urls loaded in the devtools are present in the netlog file

**Comment by huettenhain** [comment]
Unfortunately, whatever I do, I cannot seem to find any SVG related strings in that network log file. I have opened VS Code with devtools open and verified that requests are being made, but after shutting it down the log file contains nothing that would indicate so. 

Notably, after removing the `Code` directory in `%APPDATA%` and the `.vscode` directory in `%USERPROFILE%` and having VS Code start over with defaults, I can no longer reproduce the problem. Unless you are extremely interested in figuring out where this bug comes from, I think it would be less time consuming for me to just remove the settings and start over fresh.

**Comment by jhhcs** [comment]
Unfortunately, I was wrong about that. On my actual workstation, the problem persists even after wiping all settings.

**Comment by jhhcs** [comment]
After some experimentation, this seems to be related to the fact that in my setup, the directory
```
%USERPROFILE%\.vscode
```
is a directory junction to a directory of the form
```
c:\backup\appdata\.vscode
```
I observed in the devtools that the image source for the various icons in the testing view was set as follows:
```
url("file:///c:/backup/appdata/.vscode/extensions/ms-python.python-2021.3.680753044/resources/dark/status-unknown.svg");
```
The image re-appeared when I changed it to the following value:
```
url("file:///c:/users/REDACTED/.vscode/extensions/ms-python.python-2021.3.680753044/resources/dark/status-unknown.svg");
```
![image](https://user-images.githubusercontent.com/58880204/112758061-be23c080-8fec-11eb-82c3-14bffc4143f4.png)

Finally, after removing the junction and copying the entire `.vscode` directory from `c:\backup\appdata\.vscode` to `%USERPROFILE%\.vscode`, all icons started to load correctly.

I hope this can help diagnose the problem.

**Comment by MetRonnie** [comment]
This is not a Windows-only issue. Reproduced on `Linux x64 3.10.0-1160.24.1.el7.x86_64`, VSCode `1.55.0`.

Devtools inspection shows the CSS background-image is trying to load
```
file:///net/home/<redacted>/<username>/.vscode/extensions/ms-python.python-2021.3.680753044/resources/dark/status-ok.svg
```
I have no problem accessing this image by pasting this in the browser.

However, the network panel in VSCode reveals most other icons are being loaded with the url `file:///home/<redacted>/<user>/.vscode/...` i.e. without the `net/` at the beginning. On this shared filesystem setup the `/net/` and `/net/home/` are functionally equivalent

**Comment by MetRonnie** [comment]
This seems to now be fixed for me on 1.57.1
